tests/pipelines/test_pipelines_common.py

```diff
@@ -1808,6 +1857,86 @@ def callback_increase_guidance(pipe, i, t, callback_kwargs):
         # accounts for models that modify the number of inference steps based on strength
         assert pipe.guidance_scale == (inputs["guidance_scale"] + pipe.num_timesteps)
 
+    def test_official_callback(self):
+        sig = inspect.signature(self.pipeline_class.__call__)
+        has_callback_step_end = "callback_on_step_end" in sig.parameters
+        has_mask_image = "has_mask_image" in sig.parameters
+        has_strength = "strength" in sig.parameters
+
+        if not (has_callback_step_end):
+            return
+
+        # create a dummy callback
+        class DummyCallback(PipelineCallback):
+            tensor_inputs = ["latents"]
+
+            def callback_fn(self, pipeline, step_index, timesteps, callback_kwargs):
+                # iterate over callback args
+                for tensor_name, tensor_value in callback_kwargs.items():
+                    # check tensor inputs match callback args
+                    assert tensor_name in self.tensor_inputs
+                return callback_kwargs
+
+        test_callback = DummyCallback(cutoff_step_ratio=0.0)
+
+        components = self.get_dummy_components()
+        pipe = self.pipeline_class(**components)
+        pipe = pipe.to(torch_device)
+        pipe.set_progress_bar_config(disable=None)
+
+        inputs = self.get_dummy_inputs(torch_device)
+        inputs["callback_on_step_end"] = test_callback
+        inputs["output_type"] = "latent"
```

Comment:
can we just do "np" here? some of the pipelines does not support "latent" but they should all support "np"
```suggestion
        inputs["output_type"] = "np"
```